src/Playwright.TestAdapter/PlaywrightSettingsXml.cs

```diff
@@ -82,16 +82,30 @@ private static T ParseXmlIntoClass<T>(XmlReader reader) where T : class, new()
             if (reader.NodeType == XmlNodeType.Element)
             {
                 var key = reader.Name;
-                reader.Read();
-                if (reader.NodeType != XmlNodeType.Text)
+                var property = classType.GetProperty(key);
+                if (property == null)
                 {
-                    Console.Error.WriteLine($"Playwright RunSettings Parsing Error: Playwright>{endTag}>{key} is not supported");
+                    Console.WriteLine($"Playwright RunSettings Parsing Error: Playwright>{endTag}>{key} is not supported");
```

Comment:
note: messages to the stderr don't get shown, hence changing it to stdout as a drive-by.

Reply:
This sounds wrong, why is it not shown?